index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<main id="main">
+  <head> 
+    <meta charset="UTF-8">
+    <title >FCC: Tribute page</title>
+   
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <h1 id = "title"> Dr. Norman Borlaug </h1>
+    <p id="title">
+      The man who saved a billion lives
+    </p>
+    <div id="img-div">
+      <img  id="image" src="tribute-page-main-image.jpg" alt="image of biologist">
+      <p id="img-caption">
+        Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.
+      </p>
+    </div><br>
+
+    
+    <h3 id="title">
+      Here's a time line of Dr. Borlaug's life:
+    </h3><br>
+    <div id="tribute-info">
```

Comment:
I think it would be better to use a section tag here. This shows that you are starting to code a new section of the page.

Reply:
- Nicely done inline comment. It is polite, clear and explanatory :+1: